src/content/docs/dropins/cart/dictionary.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Cart dictionary
+description: Learn how to customize the default dictionary for the cart drop-in component.
+---
+
+import CodeInclude from '@components/CodeInclude.astro';
+import CodeImport from '@components/CodeImport.astro';
+import dictionary from '@dropins/storefront-cart/i18n/en_US.json.d.ts?raw';
+
+The default cart dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the cart drop-in component.
+
+## Default keys and values
+
+<CodeInclude code={dictionary} title="en_US.json" lang="js" />
+
+## Customizing the dictionary
+
+To customize the dictionary, you can create a new dictionary file and import it into your project. The new dictionary file should contain only the keys you want to override. The keys in the new dictionary file will be merged with the default dictionary.
+
+For example, to customize the "Continue Shopping" button text, create a new dictionary file with the following content:
+
+```typescript
+// src/i18n/en_US.json
+export default {
+  cart: {
+    continueShopping: 'Back to shopping',
+  },
+};
+```
+
+Then import the new dictionary file into your project:
+
+```typescript
+import customDictionary from '../i18n/en_US.json.d.ts';
+
+const dictionary = {
+  ...defaultDictionary,
+  ...customDictionary,
+};
+```
+
+You can then use the `dictionary` object in your components to access the customized values.
+
+## Dictionary keys
```

Comment:
What's the relationship between this and the "default keys and values" described on line 12?

Reply:
This needs to go away too. Or be replaced with something consistent across the different drop-ins.